ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -3086,7 +3086,11 @@ FEDataManager::collectActivePatchElements(std::vector<std::vector<Elem*> >& acti
                                           const int finest_elem_ln)
 {
     // Get the necessary FE data.
-    const MeshBase& mesh = d_fe_data->d_es->get_mesh();
+    //
+    // We once relied on libMesh not correctly implementing iterator constness
+    // correctly. Here, we don't need to change the mesh, but we use Elem *s
```

Comment:
Could we use `const Elem*`s here?

Reply:
Yes, but it would require that we change some more APIs - we would have to change the type of `getActivePatchElementMap()` and any data structure that presently stores `Elem`s for partitioning purposes.